drivers/spi/Kconfig

```diff
@@ -176,6 +176,46 @@ config SPI_6_OP_MODES
 
 endif # SPI_6
 
+config SPI_7
```

Comment:
that reminds me we will really need to fix this: removing instance knowledge from Kconfig.